composer.json

```diff
@@ -18,7 +18,7 @@
     "require": {
         "php": ">=5.5",
         "guzzlehttp/guzzle": "~5.3|~6.0.1|~6.1",
-        "guzzlehttp/psr7": "~1.0",
+        "guzzlehttp/psr7": "~1.3.1",
```

Comment:
`~1.3.1` is probably not what you really want as it will not allow 1.4 anymore. You should consider using `^1.3.1`